com.eKitabu.appdata.xml

```diff
@@ -0,0 +1,30 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<component type="desktop">
+  <id>com.ekitabu.eKitabu.desktop</id>
+  <name>eKitabu</name>
+  <project_license>LicenseRef-proprietary</project_license>
+  <developer_name>eKitabu</developer_name>
+  <summary>eReader for Desktop</summary>
+  <metadata_license>CC0-1.0</metadata_license>
+  <url type="homepage">https://ekitabu.com</url>
+  <url type="help">https://support.ekitabu.com/helpdocuments/ekitabu-e-reader-for-windows</url>
+  <description>
+    <p>eKitabu is a multifaceted ebook reader. Add EPUB books easily from your device or online feeds. eKitabu e-reader has a clean viewer that allows you to set text size, change themes, bookmark pages, highlight text, and add notes to a page or selected text.</p>
+  </description>
+  <screenshots>
+    <screenshot type="default">
+      <image type="source">https://ekitabumediastorage.blob.core.windows.net/appscreenshots/Screenshot%202022-11-25%20at%2011.09.22.png</image>
+    </screenshot>
+  </screenshots>
+  <categories>
+    <category>education</category>
```

Comment:
This is not a valid category. As [documented in the AppStream specification](https://www.freedesktop.org/software/appstream/docs/chap-Metadata.html#tag-categories):

> A list of valid category names can be found in the [Freedesktop menu specification](https://specifications.freedesktop.org/menu-spec/latest/apa.html).